### DISCUSSION TITLE
Moes zigbee gu10 bulbs and ikea pir on domoticz

### DISCUSSION THREAD

**Discussion by Barry40** [Koenkk/zigbee2mqtt#19089] [Devices]
Hi all,
I'm a happy user of z2m, but...
I have a lot moes gu10 zigbee lamps that not working like they should so.
I use domoticz, and everything was working before in our old house with ledvance devices, but we just moved to our new home and so i started again from scratch..
Some moes lights goes on (after motion detected,) but directly to off again.
So, in short it is blinking a second, then off directly.
I have to send a command twice to turn (and let the light) on, this time it stays on.
What can be the issue?
All settings of all devices are the same, some devices works good, some others does not.
Its a weird thing going on.
Also its not all the time, aometimes they work like expected.

I did a reboot, new adoption of the devices and did some testing with or without caching, or dimlevel instead of ON, but it didnt help.
In z2m is seen that the light has to be on, but it is not. When send the command again, the light goes on and then it stays on..

I'm using the sonoff usb zigbee coordinator.

I hope someone can help me with this! 

Many thanks!
Barry

**Comment by Razor5284** [comment]
Hi, experiencing the same issue now. Did you manage to find a fix?
